.github/workflows/linting.yml

```diff
@@ -3,28 +3,26 @@ name: Linting
 on:
   push:
     branches: [ development ]
-  pull_request:
-    branches: [development]    
+  pull-request:
+    branches: [development]
+
 
-  
 jobs:
   linting:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
-        
-    
-      - name: Install dependencies
-        run: |
-          sudo apt install clang-format
-          
+
 # Check code meets Google C++ style guide https://google.github.io/styleguide/cppguide.html
       - name: Run linting
-        run: | 
-          find ../ -regex '.*\.\(cc\|h\)' -not -iname '*.in.h' | xargs -I{} -P 10 clang-format -i -style=file {}; git diff
-          
+        uses: DoozyX/clang-format-lint-action@v0.16.2
+        with:
+          source: '.'
+          extensions: 'h,cc'
```

Comment:
The previous version excluded `.in.h` files, does this do the same?

Reply:
Yes. In fact, it only includes `cc, h` files.